scipy/stats/stats.py

```diff
@@ -4095,20 +4097,105 @@ def fisher_exact(table, alternative='two-sided'):
     See Also
     --------
     chi2_contingency : Chi-square test of independence of variables in a
-        contingency table.
+        contingency table.  This can be used as an alternative to
+        `fisher_exact` when the numbers in the table are large.
     barnard_exact : Barnard's exact test, which is a more powerful alternative
         than Fisher's exact test for 2x2 contingency tables.
 
     Notes
     -----
+    *Null hypothesis and p-values*
+
+    The null hypothesis is that the input table is from the hypergeometric
+    distribution with parameters (as used in `scipy.stats.hypergeom`)
+    ``M = a + b + c + d``, ``n = a + b`` and ``N = a + c``, where the
+    input table is ``[[a, b], [c, d]]``.  This distribution has support
+    ``max(0, N + n - M) <= x <= min(N, n)``, or, in terms of the values
```

Comment:
How about rendering this as latex blocks?

```suggestion
    :math:`\max(0, N + n - M) \le x \le \min(N, n)`, or, in terms of the values
```

Reply:
I'd prefer these as they are to be honest. It looks okay as code in html, and the plain text version is way more readable than in LaTeX format.

Reply:
> It looks okay as code in html, and the plain text version is way more readable than in LaTeX format.

Oh, OK. I thought using LaTeX can help with how they look in HTML, but didn't think of readability in REPL. If that's something we should look out for, I agree this makes more sense as it is. Thanks!

Reply:
Yeah, often the code/pseudo-code is awkward, and then it makes total sense to use LaTeX in the Notes section. But if it looks good as code, then I'd say prefer that.

Reply:
Also in the editor - my brain needs more time to parse `\le` than `<` :)